src/Cli/Exporter.cs

```diff
@@ -23,7 +23,7 @@ public static void Export(ExportOptions options, ILogger logger, FileSystemRunti
             if (!TryGetConfigFileBasedOnCliPrecedence(loader, options.Config, out string runtimeConfigFile))
             {
                 logger.LogError("Failed to find the config file provided, check your options and try again.");
-                return;
+                return -1;
```

Comment:
Can we define the value -1 as a const error code somewhere?

Reply:
Yes, introduced constants folder with CliReturnCode class with two constants CliReturnCode.GENERAL_ERROR and CliReturnCode.SUCCESS